CLAUDE.md

```
@@ -0,0 +1,62 @@
+  updateBoard
+  deleteBoard
+  ```
+
+- auth 관련은 접두사 생략 (login, logout, signup)
+
+### 4. TanStack Query
+
+- path: hooks/queries/{domain}/use{Domain}.ts
+
+- pattern:
+  ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)